### PR TITLE
Show confirmation modal when clear messages is clicked in topics list

### PR DESCRIPTION
**Pull request by simonyandev**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
Open confirmation modal when clear messages is clicked and refetch to update data.

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [x] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-react-app/src/components/Topics/List/List.tsx [hunk 1]
```diff
@@ -200,6 +196,11 @@ const List: React.FC<TopicsListProps> = ({
         setRecreateTopicConfirmationVisible,
       ] = React.useState(false);
 
+      const [
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Topics/List/List.tsx:199]: maybe we can use useModal hook here , cause it is more declarative

File: kafka-ui-react-app/src/components/Topics/List/List.tsx [hunk 2]
```diff
@@ -208,6 +209,8 @@ const List: React.FC<TopicsListProps> = ({
 
       const clearTopicMessagesHandler = React.useCallback(() => {
         clearTopicMessages(clusterName, name);
+        fetchTopicsList(fetchTopicsListParams);
+        setClearMessagesConfirmationVisible(false);
       }, [name]);
```
Comment (Mgrdich) [inline kafka-ui-react-app/src/components/Topics/List/List.tsx:214]: shouldn't fetchTopicsListParams be in the dependency ?

Comment (simonyandev) [reply kafka-ui-react-app/src/components/Topics/List/List.tsx:214]: Right, missed that.

**Review by workshur — APPROVED** [review]

File: kafka-ui-react-app/src/components/Topics/List/List.tsx (hunk 1 above)
Comment (simonyandev) [reply kafka-ui-react-app/src/components/Topics/List/List.tsx:199]: Or maybe I'll refactor all of them (in this component) with this PR.

**Review by Mgrdich — APPROVED** [review]

**Review by rAzizbekyan — APPROVED** [review]

**Review by lazzy-panda — APPROVED** [review]

**Pull request merged**